src/core/requests.ts

```diff
@@ -59,18 +87,24 @@ namespace Requests {
   };
 
   export const autocompleteSuggestions = (config: AppConfig): QueryTimeAutocompleteConfig =>
-    Requests.chain(config.autocomplete.defaults.suggestions, {
-      language: Autocomplete.extractLanguage(config),
-      numSearchTerms: Configuration.extractAutocompleteSuggestionCount(config),
-      numNavigations: Configuration.extractAutocompleteNavigationCount(config),
-      sortAlphabetically: Configuration.isAutocompleteAlphabeticallySorted(config),
-      fuzzyMatch: Configuration.isAutocompleteMatchingFuzzily(config)
-    }, config.autocomplete.overrides.suggestions);
-
-  export const autocompleteProducts = (state: Store.State): QueryTimeProductSearchConfig => {
+    Requests.chain(
+      Configuration.autocompleteSuggestionsDefaults(config),
+      normalizeToFunction({
```

Comment:
Move this object out and name it for readability and consistency with the other functions.